### PR TITLE
Property Component

### PR DESCRIPTION
**Pull request by slashmili**
In level 10 we wrap the data into Property Component

### CODE REVIEW

**Comment by nonninz** [comment]
Is the idea that all fields in `AutoApi.*State` modules will be `AutoApi.PropertyComponent` struct?

**Comment by slashmili** [comment]
@nonninz Yes. That is the idea, we are doing the same in other SDKs. Do you have something else in mind?

**Comment by nonninz** [comment]
> @nonninz Yes. That is the idea, we are doing the same in other SDKs. Do you have something else in mind?

Nono, that's fine for me. Even better, as that allows us (hopefully) to move there some of the parsing code, away from `AutoApi.State` which was getting a bit too big :)

File: lib/auto_api/property_component.ex [hunk 1]
```diff
@@ -0,0 +1,201 @@
+# AutoAPI
+# Copyright (C) 2018 High-Mobility GmbH
```
Comment (scudelletti) [inline lib/auto_api/property_component.ex]: 2019? 😄

**Pull request merged**